README.md

```diff
@@ -257,3 +256,118 @@ We can now scale Kibana and Fluentd back up to their previous state.  Since Flue
+```
+
+After that, you can verify that fluentd has been able to start reading in log files
+by checking the contents of `/var/log/node.log.pos` and `/var/log/es-containers.log.pos`.
+node.log.pos will keep track of the placement in syslog log files and es-containers.log.pos
+will keep track of the placement in the docker log files (/var/log/containers).
+
+### Elasticsearch
+
+Elasticsearch will have more logs upon start up than Fluentd and it will give you
+more information such as how many indices it recovered upon starting up.
+```
+[2016-02-19 20:40:42,983][INFO ][node                     ] [Volcana] version[1.5.2], pid[7], build[62ff986/2015-04-27T09:21:06Z]
+[2016-02-19 20:40:42,983][INFO ][node                     ] [Volcana] initializing ...
+[2016-02-19 20:40:43,546][INFO ][plugins                  ] [Volcana] loaded [searchguard, openshift-elasticsearch-plugin, cloud-kubernetes], sites []
+[2016-02-19 20:40:46,749][INFO ][node                     ] [Volcana] initialized
+[2016-02-19 20:40:46,767][INFO ][node                     ] [Volcana] starting ...
+[2016-02-19 20:40:46,834][INFO ][transport                ] [Volcana] bound_address {inet[/0:0:0:0:0:0:0:0:9300]}, publish_address {inet[/172.17.0.1:9300]}
+[2016-02-19 20:40:46,843][INFO ][discovery                ] [Volcana] logging-es/WJSOLSgsRuSe183-LE0WwA
+SLF4J: Class path contains multiple SLF4J bindings.
+SLF4J: Found binding in [jar:file:/usr/share/elasticsearch/plugins/openshift-elasticsearch-plugin/slf4j-log4j12-1.7.7.jar!/org/slf4j/impl/StaticLoggerBinder.class]
+SLF4J: Found binding in [jar:file:/usr/share/elasticsearch/plugins/cloud-kubernetes/slf4j-log4j12-1.7.7.jar!/org/slf4j/impl/StaticLoggerBinder.class]
+SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
+SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
+[2016-02-19 20:40:49,980][INFO ][cluster.service          ] [Volcana] new_master [Volcana][WJSOLSgsRuSe183-LE0WwA][logging-es-4sbwcpvw-1-d5j4y][inet[/172.17.0.1:9300]], reason: zen-disco-join (elected_as_master)
+[2016-02-19 20:40:50,005][INFO ][http                     ] [Volcana] bound_address {inet[/0:0:0:0:0:0:0:0:9200]}, publish_address {inet[/172.17.0.1:9200]}
+[2016-02-19 20:40:50,005][INFO ][node                     ] [Volcana] started
+[2016-02-19 20:40:50,384][INFO ][gateway                  ] [Volcana] recovered [0] indices into cluster_state
+```
+At this point, you know that ES is currently up and running.
+
+If you see a stack trace like the following from `com.floragunn.searchguard.service.SearchGuardConfigService` you can ignore it.  It is due
+to the Searchguard plugin not gracefully handling the ES service not being up and ready at the time it is querying for its configurations.
+While you can ignore stack traces from searchguard it is important to still look for stack traces due that explain why ES may not have started up,
```

Comment:
While you can ignore stack traces from searchguard it is important to still review them to determine why ES may not have started up, especially if the searchguard stack trace is repeated multiple times: